cmd/root.go

```diff
@@ -137,7 +129,7 @@ func detectNewVersion() {
 		return
 	}
 
-	blobBody := blobStream.Body(azblob.RetryReaderOptions{MaxRetryRequests: downloadMaxTries})
+	blobBody := blobStream.Body(azblob.RetryReaderOptions{MaxRetryRequests: ste.MaxRetryPerDownloadBody})
```

Comment:
Minor: Weren't we trying to get rid of the download options (per ste/xfer.go:40)?

Reply:
hmmm I don't have any context on that comment unfortunately